java/google/registry/rde/imports/XjcToDomainResourceConverter.java

```diff
@@ -293,16 +238,34 @@ private static HistoryEntry createHistoryEntry(XjcRdeDomain domain) {
   }
 
   /** Converts {@link XjcRdeDomainTransferDataType} to {@link TransferData}. */
-  private static TransferData convertDomainTransferData(XjcRdeDomainTransferDataType data) {
+  private static TransferData convertDomainTransferData(XjcRdeDomainTransferDataType data,
+      DateTime domainExpiration) {
     if (data == null) {
       return TransferData.EMPTY;
     }
+    // If the transfer is pending, calculate the number of years to add to the domain expiration
+    // on approval of the transfer.
+    TransferStatus transferStatus = TRANSFER_STATUS_MAPPER.xmlToEnum(data.getTrStatus().value());
+    // Get new expiration date
+    DateTime newExpirationTime = domainExpiration;
+    if (transferStatus == TransferStatus.PENDING) {
+      // Default to domain expiration time plus one year if no expiration is specified
```

Comment:
Do you anticipate getting deposits where there is a pending transfer yet no exDate specified?  I'll admit the specs are fairly vague about exDate and do indicate it's optional, but I'm curious to know whether this special-casing is in response to deposits you've seen "in the wild" or just a contingency.